ocaml/idl/autogen/dune

```diff
@@ -0,0 +1,6 @@
+(alias
```

Comment:
this is an alias with a name and dependency, but where is the action?

Reply:
No action, just using `source_tree` to include all the files that will be generated in here. We have a few more dune files with rules like this.